.github/workflows/dev-build.yml

```
@@ -0,0 +1,99 @@
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [linux]
+        goarch: [amd64, arm64]
+    env:
+      OUTPUT_BINARY: speedtest-ex
+      GO_VERSION: 1.23.4
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)